docs/reference/intro.asciidoc

```diff
@@ -155,81 +184,146 @@ You can use a combination of dynamic and explicit mapping on the same index.
+
+[TIP]
+====
+If you're interested in data ingestion pipelines for time-series data, use the decision tree in the {cloud}/ec-cloud-ingest-data.html#ec-data-ingest-pipeline[Elastic Cloud docs] to understand your options.
+====
+
+[discrete]
+[[es-ingestion-overview-general-content]]
+==== General content
+
+For general content, you have the following options for adding data to {es} indices:
+
+* <<docs,API>>: Use the {es} <<docs,Document APIs>> to index documents directly, using the Dev Tools {kibana-ref}/console-kibana.html[Console] in {kib}, cURL, or an https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} programming language client].
++
+[TIP]
+====
+Once you get past the very basics, you'll want to use the <<docs-bulk,Bulk API>> for efficiency.
+====
+* {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[File upload]: Use the {kib} File Uploader to upload and index CSV, JSON, and log files.
+* {kibana-ref}/connect-to-elasticsearch.html#_add_sample_data[Sample data]: Load sample data sets into your {es} cluster using {kib}.
+* {enterprise-search-ref}/crawler.html[Web Crawler]: Extract and index web page content into {es} documents.
+* {enterprise-search-ref}/connectors.html[Connectors]: Sync data from various third-party data sources to create searchable, read-only replicas in {es}.
+
 // New html page
 [[search-analyze]]
-=== Search and analyze
+=== Search and analyze data
 
-While you can use {es} as a document store and retrieve documents and their
-metadata, the real power comes from being able to easily access the full suite
-of search capabilities built on the Apache Lucene search engine library.
+You can use {es} as a basic document store to simply retrieve documents and their
+metadata.
+However, the real power of {es} comes from its advanced search and analytics capabilities.
 
 {es} provides a simple, coherent REST API for managing your cluster and indexing
-and searching your data. For testing purposes, you can easily submit requests
-directly from the command line or through the Developer Console in {kib}. From
-your applications, you can use the
-https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} client]
-for your language of choice: Java, JavaScript, Go, .NET, PHP, Perl, Python
-or Ruby.
+and searching your data.
+For testing purposes, you can easily submit requests
+directly from the command line or through the Dev Tools {kibana-ref}/console-kibana.html[Console] in {kib}.
+From your applications, you can use the
+https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} clients]
+in your programming language of choice.
+
+<<query-dsl, Query DSL>> is the primary query language for {es} today.
+It's a full-featured JSON-style query language that allows you to perform complex searches and aggregations.
+We'll be using the Query DSL for most of our examples.
```

Comment:
couple things

1. connect the API + QL.
2. introduce the concept of query languages broadly

3. "we" / "our" is vague here
```suggestion
<<query-dsl, Query DSL>> is the primary query language for {es} today.
It's a full-featured JSON-style query language that allows you to perform complex searches and aggregations.
This documentation uses Query DSL for most query examples.
```